src/ontology/mondo-edit.obo

```diff
@@ -620039,6 +620054,7 @@ id: has_material_basis_in_germline_mutation_in
 name: has material basis in germline mutation in
 xref: RO:0004003
 is_transitive: true
+is_a: RO:0004018 ! is basis for realizable
```

Comment:
@MeeSiing did you make this change?

Reply:
@MeeSiing, this should be removed if you did not add it. 
I can take care of it.